duo_wordpress.php

```diff
@@ -378,6 +383,15 @@ function duo_settings_xmlrpc() {
         echo "<input id='duo_xmlrpc' name='duo_xmlrpc' type='checkbox' value='off' $val /> Yes<br />";
         echo "Using XML-RPC bypasses two-factor authentication and makes your website less secure. We recommend only using the WordPress web interface for managing your WordPress website.";
     }
+    function duo_settings_rest() {
+        $val = '';
+        if(duo_get_option('duo_rest', 'off') == 'off') {
```

Comment:
But at this point, we say "If an option hasn't been set, display as checked (REST allowed) by default." The two should have the same behavior. If we are not allowing REST by default (per my other comment), then we should not toggle the checkbox to show that we are. Given that this option specifically states "makes your website less secure," I would think we would want to show this option as unchecked by default.

Please correct me if I'm misunderstanding something. Thanks!